store/keys_test.go

```diff
@@ -0,0 +1,79 @@
+package store
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
ditto. Malformed. Please run `make fmt`. Also, don't mix `assert` with `require` -- only use the latter.